qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -83,19 +64,24 @@ Required fields for EBI submission
-| ``collection_timestamp``         | ``mm/dd/yy hh:mm``   | Date and time with time in 24-hour format in which the sample was collected.                                                                        |
-+----------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``physical_specimen_location``   | free text            | Where you would go to find physical sample or DNA, regardless of whether it is still available or not.                                              |
-+----------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``taxon_id``                     | integer              | NCBI's taxon id for the sample. Note, for amplicon sequencing, this is the taxonomy id for the metagenome being targeted, not the host taxonomy id. |
-+----------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``description``                  | free text            | Description of the sample.                                                                                                                          |
-+----------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``scientific_name``              | free text            | NCBI's scientific name for the provided taxon ID. Note, the name of the metagenome, not the host scientific name.                                   |
-+----------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
++----------------------------------+-----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field name                       | Format                | Description                                                                                                                                         |
++==================================+=======================+=====================================================================================================================================================+
+| ``collection_timestamp``         | ``mm/dd/yy hh:mm:ss`` | The time stamp (preferred) of when the sample was collected. Several format are accepted.                                                           |
```

Comment:
Actually, according to the test the years need 4 digits, correct? so it should be `yyyy` for all of these.